### PR TITLE
cdc: update debezium protocol 

### PR DESCRIPTION
**Pull request by wk989898**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add key field for Debezium protocol. see https://github.com/pingcap/tiflow/issues/11652

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-debezium.md [hunk 1]
```diff
@@ -29,7 +29,38 @@ Debezium 输出格式中包含当前行的 Schema 信息，以便下游消费者
...
+    }
+}
+```
+
+Key 中的 fields 只包含主键或唯一索引列。字段解释如下：
+
+| 字段      | 类型   | 说明                                                                      |
+|:----------|:-------|:-------------------------------------------------------------------------|
+| payload   | JSON | 主键或唯一索引列的信息。每个 field 的 key 和 value 分别为列名和当前值  |
+| schema.fields     | JSON   |  payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
```
Comment (hongyunyan) [inline ticdc/ticdc-debezium.md:61]: schema name 和 optional 的字段的意思是否需要解释一下呢？

**Review by hongyunyan — APPROVED** [review]
rest is LGTM

File: ticdc/ticdc-debezium.md (hunk 1 above)
Comment (wk989898) [reply ticdc/ticdc-debezium.md:61]: 保持和value的说明一致，没有特别解释

File: ticdc/ticdc-debezium.md [hunk 2]
```diff
@@ -29,7 +29,41 @@ Debezium 输出格式中包含当前行的 Schema 信息，以便下游消费者
...
+| schema.name     | 字符串   |  schema 的名称，格式为"{cluster}.{schema}.{table}.Key"     |
+| schema.optional     | 布尔值   |  optional 为 true 时表示该 field 为选填项   |
+| schema.type     | 字符串   |  表示该 field 的类型   |
```
Comment (qiancai) [inline ticdc/ticdc-debezium.md:64]:
```suggestion
| schema.name     | 字符串   |  schema 的名称，格式为 `"{cluster-name}.{schema-name}.{table-name}.Key"`     |
| schema.optional     | 布尔值   |  为 true 时表示该 field 为选填项   |
| schema.type     | 字符串   |  表示该 field 的数据类型   |
```

File: ticdc/ticdc-debezium.md [hunk 3]
```diff
@@ -126,6 +160,9 @@ TiCDC 会把一个 DML Event 编码成如下格式：
 | payload.source.db     | 字符串   | 事件发生的数据库的名称                    |
 | payload.source.table     | 字符串  |  事件发生的数据表的名称                    |
 | schema.fields     | JSON   |  payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
+| schema.name     | 字符串   |  schema 的名称，格式为"{cluster}.{schema}.{table}.Envelope"     |
+| schema.optional     | 布尔值   |  optional 为 true 时表示该 field 为选填项   |
```
Comment (qiancai) [inline ticdc/ticdc-debezium.md:164]:
```suggestion
| schema.name     | 字符串   |  schema 的名称，格式为 `"{cluster}.{schema}.{table}.Envelope"`     |
| schema.optional     | 布尔值   |  optional 为 `true` 时表示该 field 为选填项   |
```

**Review by qiancai — APPROVED** [review]

File: ticdc/ticdc-debezium.md [hunk 4]
```diff
@@ -29,7 +29,41 @@ Debezium 输出格式中包含当前行的 Schema 信息，以便下游消费者
 
 ### DML Event
 
-TiCDC 会把一个 DML Event 编码成如下格式：
+TiCDC 会将一个 DML 事件转换为一个 kafka 事件，其中事件的 key 和 value 都按照 Debezium 协议进行编码。
```
Comment (Oreoxmt) [inline ticdc/ticdc-debezium.md:32]:
```suggestion
TiCDC 会将一个 DML 事件转换为一个 Kafka 事件，其中事件的 key 和 value 都按照 Debezium 协议进行编码。
```

File: ticdc/ticdc-debezium.md [hunk 5]
```diff
@@ -29,7 +29,41 @@ Debezium 输出格式中包含当前行的 Schema 信息，以便下游消费者
...
+            }
+        ],
+        "name": "default.test.t2.Key",
+        "optional": false,
+        "type": "struct"
+    }
+}
+```
+
+Key 中的 fields 只包含主键或唯一索引列。字段解释如下：
```
Comment (Oreoxmt) [inline ticdc/ticdc-debezium.md:56]:
```suggestion
Key 中的字段只包含主键或唯一索引列。字段解释如下：
```
fields 可能误以为是 `schema.fields`

File: ticdc/ticdc-debezium.md [hunk 6]
```diff
@@ -29,7 +29,41 @@ Debezium 输出格式中包含当前行的 Schema 信息，以便下游消费者
...
+
+| 字段      | 类型   | 说明                                                                      |
+|:----------|:-------|:-------------------------------------------------------------------------|
+| payload   | JSON | 主键或唯一索引列的信息。每个 field 的 key 和 value 分别为列名和当前值  |
+| schema.fields     | JSON   |  payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
+| schema.name     | 字符串   |  schema 的名称，格式为 `"{cluster-name}.{schema-name}.{table-name}.Key"`     |
+| schema.optional     | 布尔值   |  optional 为 `true` 时表示该 field 为选填项   |
+| schema.type     | 字符串   |  表示该 field 的数据类型   |
```
Comment (Oreoxmt) [inline ticdc/ticdc-debezium.md:64]:
```suggestion
| payload   | JSON | 主键或唯一索引列的信息。每个字段的 key 和 value 分别为列名和当前值  |
| schema.fields     | JSON   |  payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
| schema.name     | 字符串   |  schema 的名称，格式为 `"{cluster-name}.{schema-name}.{table-name}.Key"`     |
| schema.optional     | 布尔值   |  optional 为 `true` 时表示该字段为选填项   |
| schema.type     | 字符串   |  表示该字段的数据类型   |
```

File: ticdc/ticdc-debezium.md [hunk 7]
```diff
@@ -126,6 +160,9 @@ TiCDC 会把一个 DML Event 编码成如下格式：
 | payload.source.db     | 字符串   | 事件发生的数据库的名称                    |
 | payload.source.table     | 字符串  |  事件发生的数据表的名称                    |
 | schema.fields     | JSON   |  payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
+| schema.name     | 字符串   |  schema 的名称，格式为 `"{cluster}.{schema}.{table}.Envelope"`     |
+| schema.optional     | 布尔值   |  optional 为 `true` 时表示该 field 为选填项   |
+| schema.type     | 字符串   |  表示该 field 的类型   |
```
Comment (Oreoxmt) [inline ticdc/ticdc-debezium.md:165]:
```suggestion
| schema.name     | 字符串   |  schema 的名称，格式为 `"{cluster}.{schema}.{table}.Envelope"`     |
| schema.optional     | 布尔值   |  optional 为 `true` 时表示该字段为选填项   |
| schema.type     | 字符串   |  表示该字段的类型   |
```

**Pull request merged**